src/store/flow/block-types/MobilePrimitives_SelectOneResponseBlockStore.ts

```diff
@@ -17,31 +17,52 @@ import {
   IResourceDefinition,
 } from '@floip/flow-runner/src/domain/IResourceResolver'
 import Vue from 'vue'
-import {defaults, find, max} from 'lodash'
+import {defaults, find, max, filter, first, get} from 'lodash'
 
 export const BLOCK_TYPE = 'MobilePrimitives\\SelectOneResponse'
 
 import { someItemsHaveValue, allItemsHaveValue, twoItemsBlank } from '../utils/listBuilder'
 
+interface IInflatedChoicesInterface {
+  exit: IBlockExit,
+  resources: IResourceDefinition
+}
+
 export const getters: GetterTree<IFlowsState, IRootState> = {
   inflatedChoices: (state, getters, rootState, rootGetters): object => {
     const currentBlock = rootGetters['builder/activeBlock']
-    let choices: {[key: string]: IResourceDefinition} = {}
-    return Object.keys(currentBlock.config.choices).reduce((memo, choiceKey): {[key: string]: IResourceDefinition} => {
-      memo[choiceKey] = rootGetters['flow/resourcesByUuid'][currentBlock.config.choices[choiceKey]]
+    let choices: { [key: string]: IInflatedChoicesInterface } = {}
+
+    return Object.keys(currentBlock.config.choices).reduce((memo, choiceKey): {[key: string]: IInflatedChoicesInterface} => {
+      const resourceUuid = currentBlock.config.choices[choiceKey]
+      memo[choiceKey] = {
+        exit: getters.blockExitFromResourceUuid(resourceUuid),
+        resources: rootGetters['flow/resourcesByUuid'][resourceUuid]
+      }
       return memo
     }, choices)
   },
+  blockExitFromResourceUuid: (state, getters, rootState, rootGetters) => (resourceUuid: string): IBlockExit => {
+    const currentBlock = rootGetters['builder/activeBlock']
+    return first(filter(currentBlock.exits, {
+      config: {
+        choiceResource: resourceUuid
+      }
+    }))
+  },
+  isInflatedChoiceBankOnKey: (state, getters) => (key): boolean => {
+    return !someItemsHaveValue(getters.inflatedChoices[key].resources.values, "value") && !get(getters.inflatedChoices[key], 'exit.semanticLabel')
```

Comment:
Is this multiple resources or a single resource? How do we get values from an array of resources? Isn't that only an attribute of a single resource?

If I understand this correctly it's probably enough to just change `resources` to `resource`

Reply:
this is a single `resource`, I'll rename it accordingly. Thanks